apps/email/index.html

```diff
@@ -32,6 +32,7 @@
   <link rel="stylesheet" type="text/css" href="shared/style/progress_activity.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/tabs.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/value_selector.css" >
+  <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" >
```

Comment:
If UX officially decided to never have the carets for either rtl or ltr cases, then we should do 3. Last I heard, it was a bit undecided, and that is why email is in its current state.

Since this is not clear, at least to me, I filed [bug 1218547](https://bugzilla.mozilla.org/show_bug.cgi?id=1218547) to track finding out from UX, so we can go with option 1 if you like. However, if you know for sure that carets are explicitly out for the LTR and RTL cases, feel free to remove them now (option 3), whichever path works best for you. We can use bug 1218547 to track any follow up in either case.